data/data/manifests/openshift/cloud-creds-secret.yaml.template

```diff
@@ -45,4 +47,9 @@ data:
   ovirt_password: {{.CloudCreds.Ovirt.Base64encodePassword}}
   ovirt_insecure: {{.CloudCreds.Ovirt.Base64encodeInsecure}}
   ovirt_ca_bundle: {{.CloudCreds.Ovirt.Base64encodeCABundle}}
+{{- else if .CloudCreds.Nutanix}}
```

Comment:
What is, or is there, a relationship between this secret and the CSI driver operator secret `ntnx-secret`?  The [enhancement](https://github.com/openshift/enhancements/blob/8e71d3860e433fa5d118f31c69825df5913da7c1/enhancements/installer/nutanix-ahv-ipi.md?plain=1#L124) references the Nutanix CSI driver operator which depends on `ntnx-secret` in the `ntnx-system` namespace.

Reply:
From my understanding, these secrets have the same contents albeit in the case of CSI driver they tend to follow a string based encoding of the values (prism-element-ip:prism-port:username:password).

Reply:
@thunderboltsid I think this the [discussion](https://github.com/openshift/cloud-credential-operator/pull/450#discussion_r800788502) you are referring to.  Do you happen to have a timeline on when the transition to moving the configuration of the ip/port to the infrastructure CR might take place and would moving the configuration to the infrastructure CR at a later time cause issues?

Reply:
We can move the IP/Port over to infra CR at a later date without any issues.